packages/backend/src/graphql/scalars/BigInt.ts

```diff
@@ -51,7 +53,7 @@ export const GraphQLBigIntConfig: GraphQLScalarTypeConfig<
     ) {
       return coerceBigIntValue(ast.value)
     }
-    return null
+    return ''
```

Comment:
:confused: This example returns `null`
https://www.apollographql.com/docs/apollo-server/schema/custom-scalars/#example-the-date-scalar

Maybe we need to throw?
https://www.apollographql.com/docs/apollo-server/schema/custom-scalars/#example-restricting-integers-to-odd-values

Reply:
Interesting. Typescript complained that it must be `string | number | bigint`. I guess in that case we should probably throw.